Fix for command 'python.runtests' not found.
